lib/setup.js

```diff
@@ -65,12 +66,12 @@ function setup_test_instance(opt, cb) {
             return function(req, res) {
                 var args = [].slice.call(arguments);
                 if (is_control_req(req)) {
-                    args.push({ target: 'http://localhost:' + control_port });
+                    args.push({ target: 'http://' + loopback + ':' + control_port });
```

Comment:
I'd refactor the url building into a function, but maybe nit-picking.

Reply:
Addressed.